src/platforms/common/session-replay/index.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Set Up Session Replay
+sidebar_order: 4000
+supported:
+  - javascript
+  - javascript.angular
+  - javascript.ember
+  - javascript.gatsby
+  - javascript.nextjs
+  - javascript.react
+  - javascript.remix
+  - javascript.svelte
+  - javascript.vue
+description: "Learn how to enable session replay in your app if it is not already set up."
+---
+
+With [session replay](/product/session-replay/), TODO.
+
+<PlatformSection supported={["javascript", "javascript.vue", "java.spring", "react-native"]} notSupported={["java", "go", "android", "ruby", "python", "dotnet", "dart", "native"]}>
+
+## Enable Session Replay
+
+<PlatformContent includePath="session-replay/enable-session-replay" />
+
+</PlatformSection>
+
+## Configure the Sample Rate
+
+In many cases the default settings will give you a good balance of debugging information when errors happen, and insight into what users are doing when errors are not captured. If you want to make changes where are two fields you may change:
+
+1. Set a baseline sample rate for all site visitors using the <PlatformIdentifier name="replays-session-sample-rate" /> option in your SDK config to a number between `0` and `1`. (For example, to send 20% of all user sessions, set <PlatformIdentifier name="replays-session-sample-rate" /> to `0.2`.)
+2. Start to capture a session replay whenever an error happened by settings <PlatformIdentifier name="replays-on-error-sample-rate" /> option in your SDK config to a number between `0` and `1`. This will capture and send up to one minute of replay data before the error happened, and continue to record the replay afterwards.
+
+The two options can both be set. If a session is not captured by the <PlatformIdentifier name="replays-session-sample-rate" />, and an error happens, then the <PlatformIdentifier name="replays-on-error-sample-rate" /> will have a chance to start sampling.
+
+<PlatformContent includePath="session-replay/configure-sample-rate" />
+
+## Verify
+
+While you're testing, set <PlatformIdentifier name="replays-session-sample-rate" /> to `1.0`, as that ensures that every user session will be sent to Sentry.
+
+Once testing is complete, **we recommend lowering this value in production** by either lowering your <PlatformIdentifier name="replays-session-sample-rate" /> value. We still recommend keeping <PlatformIdentifier name="replays-on-error-sample-rate" /> set to `1.0`.
+
```

Comment:
Should we add another subsection here (e.g. `## Advanced Configuration` or something similar) and link to the Readme for the time being? This way, users can be made aware that there are additional configuration options.